lib/fluent/plugin/in_opensearch.rb

```diff
@@ -343,14 +343,18 @@ def update_retry_state(error=nil)
           @retry = retry_state(@retry_randomize)
         end
         @retry.step
+        if error.message.include?('EOFError (EOFError)')
+          log.error("Restart plugin because hit error #{error.message}")
+          exit(1)
```

Comment:
exit(1) caused entire termination. This is brutal manner for handling this exception. :(